getting-started/dc2-guide.rst

```diff
@@ -0,0 +1,182 @@
+   subsets:
+     singleFrameAp:
+     subset:
+       - isr
+       - characterizeImage
+       - calibrate
+       - consolidateVisitSummary
+     description: >
+       Tasks to run for single frame processing that are necessary to use the good seeing selector to build coadds for use as difference imaging templates.
+
+This example pipeline imports a pipeline from :py:mod:`lsst.ap.pipe` you may `view on GitHub <https://github.com/lsst/ap_pipe/blob/master/pipelines/ApTemplate.yaml>`__.
+There are some special configurations concerning reference catalogs that must be set for this camera and/or dataset, so the example pipeline above lists the ``calibrate`` task explicitly to add custom configurations.
+
+To run this example pipeline, save it as ``ApTemplate-DC2.yaml``, choose an appropriate output collection name, and run
+
+.. prompt:: bash
+
+   pipetask run -j 12 -b /repo/dc2 -d "band='g' AND skymap='DC2' AND tract=3829" -i 2.2i/defaults/test-med-1 -o u/USERNAME/OUTPUT-COLLECTION-1 -p ApTemplate-DC2.yaml#singleFrameAp --register-dataset-types
+
+This will take some time, but when it's done, you should have calibrated exposures and a visit summary table ready for making warps, selecting the best seeing visits, and assembling coadds for use as difference imaging templates.
+To continue, run:
+
+.. prompt:: bash
+
+   pipetask run -j 12 -b /repo/dc2 -d "skymap='DC2' AND tract=3829 AND patch=47" -i u/USERNAME/OUTPUT-COLLECTION-1 -o u/USERNAME/OUTPUT-COLLECTION-2 -p ApTemplate-DC2.yaml#makeTemplate --register-dataset-types
+
+This will also take some time.
+When it is complete, you should have good seeing coadds covering the entirety of patch 47 in tract 3829 for multiple bands and be ready to run the rest of the AP Pipeline (namely difference imaging and source association).
+
+Performing difference imaging to make an APDB
+---------------------------------------------
+
+This next step uses a second pipeline, which effectively includes :py:mod:`lsst.pipe.tasks.imageDifference.ImageDifferenceTask`, :py:mod:`lsst.ap.association.TransformDiaSourceCatalogTask`, and :py:mod:`lsst.ap.association.DiaPipelineTask`.
+
+.. prompt:: yaml
+
+   description: An AP pipeline for difference imaging with LsstCam-imSim
+
+   instrument: lsst.obs.lsst.LsstCamImSim
+   imports:
+   - location: ApTemplate-DC2.yaml
+     exclude:  # These tasks are not necessary, as we already have templates
+       - consolidateVisitSummary
+       - selectGoodSeeingVisits
+       - makeWarp
+       - assembleCoadd
+   - location: $AP_PIPE_DIR/pipelines/ApPipe.yaml
+     exclude:  # These tasks come from the ApTemplate-DC2 pipeline instead
+       - isr
+       - characterizeImage
+       - calibrate
+
+This difference imaging pipeline uses the good seeing templates we built and treats all the DP0 defaults as input "science" images.
+
+**This is a two-step process.**
+First, create an empty sqlite APDB:
```

Comment:
Please add a comment that the APDB must be empty, so they must make a new one if they need to re-run the pipeline for any reason.